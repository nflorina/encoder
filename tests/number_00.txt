1111
121142
-13371317
-17841
-13643
167994
196
-153
17542
1813779
13
-18
12
-1461737
-1994
11795
1559351
1718
-13215766
-12923729
1
END
