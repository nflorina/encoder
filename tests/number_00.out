0 0 20
11114212111317133711784136439941679611537542191813773118211461737199495117935115581711321576612923729
11114212111317133711784136439941679611537542191813773118211461737199495117935115581711321576612923729
